syft/frameworks/torch/fl/dataset.py

```diff
@@ -253,6 +253,11 @@ def workers(self):
 
         return list(self.datasets.keys())
 
+    def get_dataset(self, worker):
```

Comment:
It's not the part of the code I know the best but do we want to have a more granular way of getting datasets?
For instance, what if a worker wants to get only one of its dataset in the FederatedDataset?

Reply:
it's not mine but I believe this implementation doesn't allow more than one dataset per worker since it uses a dict to store them, if it has {bob:ptr1, alice:ptr2} and you add alice:ptr3 it will replace ptr2 not add it